docs/usage.md

```diff
@@ -29,6 +29,20 @@ system:
 sudo curl -L --output /usr/local/bin/pie https://github.com/php/pie/releases/latest/download/pie.phar && sudo chmod +x /usr/local/bin/pie
 ```
 
+### Docker installation
+
+Pie is published as binary-only Docker image, so you can install it easily during build:
+
+```Dockerfile
+FROM php:8
+
+COPY --form=ghcr.io/php/pie:latest /pie /usr/bin/pie
+```
+
+Instead of `latest` you can also use explicit versions like `x.y.z`, `x.y` or `x`, depending on stability level you want to achieve.
+
+Each image contains `/pie.asc` file too, so you can verify GPG signature of bundled PHAR file (`/pie`).
```

Comment:
The /pie.asc is exactly as (un)trustworthy as the binary itself, because it is downloaded from the same location.

Reply:
What do you propose? Removing this completely and if anyone wants to verify it, should download ASC file from Github? This would be tricky with `X` and `X.Y` version binding, because you will download dynamic version.

Reply:
Hm; yes, I see Tim's point; it probably makes sense to remove the signature from the image and to point people to the GH releases list (which is where the signature is canonically kept).

Reply:
I removed `asc` file from the image and changed usage docs.